parameters/Cargo.toml

```
@@ -93,8 +93,8 @@ version = "0.3.64"
 features = [ "XmlHttpRequest" ]
 optional = true
 
-[target."cfg(not(target_family = \"wasm\"))".dependencies.curl]
-version = "0.4.43"
+[target.'cfg(all(not(target_family = "wasm"), not(target_env = "sgx")))'.dependencies]
+curl = "0.4.43"
```

```suggestion
[target.'cfg(all(not(target_family = "wasm"), not(target_env = "sgx")))'.dependencies.curl]
version = "0.4.43"
```

We should keep the standard dependency format.